Browser/Browser.aspx.cs

```diff
@@ -2067,6 +2202,18 @@ private void SetLanguage()
                  Localization.GetString("SortDescending.Help", ResXFile, LanguageCode));
             SortDescending.ToolTip = Localization.GetString("SortDescending.Help", ResXFile, LanguageCode);
 
+            SortByDateAscending.Text = string.Format(
+                 "<img src=\"Images/AscendingArrow.png\" alt=\"{0}\" title=\"{1}\" />Date",
```

Comment:
@along-engage this _Date_ text also needs to come from a resource file

Reply:
@along-engage you got the second one, still need to fix this one.  Thanks!

Reply:
@bdukes My mistake, this has been changed as well!